lib/dai-plugin-mcd/test/CdpTypeService.spec.js

```diff
@@ -1,31 +1,22 @@
 import { createCurrencyRatio } from '@makerdao/currency';
 import { mcdMaker, setupCollateral } from './helpers';
 import { ServiceRoles } from '../src/constants';
-import { ETH, MDAI, MKR, USD, COL1 } from '../src';
+import { ETH, MDAI, MKR, USD, REP } from '../src';
 const { CDP_MANAGER, CDP_TYPE, QUERY_API } = ServiceRoles;
 
 let maker, service;
 
 // these CDP types should be available to the Maker instance because
 // of the configuration passed into it (see test/helpers.js)
-const scenarios = [['ETH-A', ETH], ['ETH-B', ETH], ['COL1-A', COL1]];
-let systemValues = {};
-
-function setSystemValues() {
-  const values = [[1000, 1.5, '2.5'], [500, 1.8, '1.5'], [800, 1.8, '2.0']];
-  let scenario;
-  for (scenario = 0; scenario < scenarios.length; scenario++) {
-    systemValues[scenarios[scenario][0]] = {
-      expectedCeiling: values[scenario][0],
-      expectedRatio: values[scenario][1],
-      expectedFee: values[scenario][2]
-    };
-  }
-}
+const scenarios = [['ETH-A', ETH], ['ETH-B', ETH], ['REP-A', REP]];
+const systemValues = {
```

Comment:
having a comment here that lists the meaning of each field in the array would help readers